src/message_constants.ts

```diff
@@ -75,6 +75,10 @@ export function WELCOME_AND_STATE_QUESTION(): string {
   return 'Welcome to the VoteAmerica Helpline! In which U.S. state are you looking to vote?\n\nMsg&data rates may apply. Reply STOP to unsubscribe.';
 }
 
+export function WELCOME_FINDING_VOLUNTEER(): string {
```

Comment:
I believe all other initial messages to voters have the msg&data rates part as well as the STOP keyword instructions. Are we okay with this one not having that?

Reply:
fixed!